asrtoolkit/data_handlers/txt.py

```diff
@@ -14,7 +14,7 @@ def format_segment(seg):
     """
     Formats a segment assuming it's an instance of class segment with text element
     """
-    return seg.text
+    return seg.formatted_text if hasattr(seg, "formatted_text") and seg.formatted_text else seg.text
```

Comment:
We can't do the code below `formatted_test = ''`?
```suggestion
    return getattr(seg, "formatted_text", seg.text)
```

Reply:
Correct, in our data handlers that initialize segment objects, we're not guaranteeing that formatted_text is non-empty even if text is populated.

Reply:
Ah, then this is a slightly better version.
```suggestion
    return seg.formatted_text if hasattr(seg, "formatted_text") and seg.formatted_text else seg.text
```